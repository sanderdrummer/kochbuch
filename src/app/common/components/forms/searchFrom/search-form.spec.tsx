import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import SearchForm from './search-form';

it('should render search form', () => {
    const wrapper = shallow((
    <SearchForm
        label="test"
        placeholder=""
        reducerName="searchUi"
        state="product"
        value="test"
        // tslint:disable-next-line:no-empty
        onSubmit={(e) => {}}
        // tslint:disable-next-line:no-empty
        resetValue={() => {}}
        // tslint:disable-next-line:no-empty
        onChanges={(e) => {}}
    />
        )
    );
    expect(toJson(wrapper)).toMatchSnapshot();    
});