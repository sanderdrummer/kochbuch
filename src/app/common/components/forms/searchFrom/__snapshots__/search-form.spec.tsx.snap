// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render search form 1`] = `
<form
  onSubmit={[Function]}
>
  <input
    onChange={[Function]}
    type="search"
    value="test"
  />
  <button
    type="submit"
  >
    search
  </button>
</form>
`;
